### DISCUSSION TITLE
Export of work time by teams

### DISCUSSION THREAD

**Discussion by lllp1** [kimai/kimai#3251] [Questions & Answers]
Hello,

I have several bands in kimai. I would like to export the recorded time of work for each team separately. Is it possible in the application?
I can't find this option. In a given month, the list of working time of all users is exported.

How to split exports into teams?

**Comment by kevinpapst** [comment]
This is not possible out of the box. 
Every user can be part of multiple times and there is no connection between a timesheet and a team.

**Reply by lllp1** [reply]
I have inaccurately described the situation.
I have employees assigned to team 1, team 2 ...
I want to export employees' working time in a month, e.g. in March, only with team 1.
Is it possible to do this?

There is a user field in the search engine. Can you add a team?
![search_timesheets](https://user-images.githubusercontent.com/99353489/162196837-cd433013-fbb8-4a09-8349-b87ce709ffe9.jpg)

**Reply by kevinpapst** [reply]
Thanks for the clarification. Unfortunately this is still not possible as this does not change the underlying issue. 

User A is in Team 1 and Team 2. 
Both teams have access to Project C.
Now User A records a time for Project C.

What happens if you filter for Team 1?
What happens if you filter for Team 2?

This is the reason why there is no team filter, it will produce wrong/unexpected results.

**Reply by lllp1** [reply]
OK thanks for the information. I did not consider such a situation.
